src/VirtoCommerce.CatalogModule.Data/Repositories/CatalogRepositoryImpl.cs

```diff
@@ -303,6 +303,27 @@ public async Task<AssociationEntity[]> GetAssociationsByIdsAsync(string[] associ
             return result;
         }
 
+        public async Task<string[]> GetAllSeoDuplicatesIdsAsync()
+        {
+            const string commandTemplate = @"
+                    WITH cte AS (
```

Comment:
Why not write there simpler:
``` SQL
SELECT Id FROM 
(
	SELECT
		Id,
		Keyword,
		StoreId,
		count (*) cnt
	FROM CatalogSeoInfo
	GROUP BY 		
		Id,
		Keyword,
		StoreId
) cte
WHERE cte.cnt > 1
```
It's clearer and ANSI-compatible

Reply:
or even (I'm not sure I understood the goal):
``` sql
SELECT DISTINCT CatalogSeoInfo.Id
FROM
(
	SELECT * FROM 
	(
		SELECT
			Keyword,
			StoreId,
			count (*) cnt
		FROM CatalogSeoInfo
		GROUP BY 		
			Keyword,
			StoreId
	) cte
	WHERE cte.cnt > 1
) cte2more
inner join CatalogSeoInfo
on CatalogSeoInfo.Keyword=cte2more.Keyword and CatalogSeoInfo.StoreId=cte2more.StoreId
```

Reply:
We are using CTE queries in many places, so I'm not sure we need to avoid them.
Also provided query is simper and more clear for me. 
It is achieving the goal for sure - it returns the ids of duplicated seo infos (by keyword and storeId). So I do not think we need to change this one.
Probably new query does not return id's of only duplicates, but all ids (the first one + duplicates).

Reply:
I don't insist.
The second query should return only for duplicates. Does not matter, anyway.